src/components/FilterBar/FilterBar.react.js

```diff
@@ -46,9 +49,14 @@ export class FilterBar extends React.Component {
             }
 
             {this.context.filterBarStore.isExportable() &&
-              <ExportResultsButton
-                filterBarActor={this.context.filterBarActor}
-              />
+              (exportAllOptions ? (
+                <ExportResultsList
+                  filterBarActor={this.context.filterBarActor}
```

Comment:
the line 54 and 55 is no use

Reply:
Fixed: https://github.com/rdytech/react-filterbar/pull/138/commits/be92fb9cb570573e819ed8f17fc9c06cfea5f458